src/services_models/services_constructor.jl

```diff
@@ -62,15 +61,15 @@ function construct_service!(
 
     for service in services
         contributing_devices =
-            services_mapping[(
-                type = typeof(service),
-                name = PSY.get_name(service),
-            )].contributing_devices
+            services_mapping[(type = SR, name = PSY.get_name(service))].contributing_devices
         if !isempty(incompatible_device_types)
             contributing_devices =
                 [d for d in contributing_devices if typeof(d) ∉ incompatible_device_types]
         end
-
+        if isempty(contributing_devices)
+            @warn("The contributing devices for service $(PSY.get_name(service)) is empty, cinsider removing the service from the system")
+            continue
```

Comment:
This service name was added above in the `names` variable.  I was asking if this inconsistency is a problem.  The validate function did not check this condition.

Also: s/cinsider/consider/